lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
the idea here was not to pollute yet another name-space - AR-JDBC managed to keep it name-spaced under its own.
probably not that relevant but a `defined? PG` can now yield surprises with AR-JDBC despite not having any PG loaded